scripts/gui/microbe_editor.mjs

```diff
@@ -249,18 +260,19 @@ function updateSelectedOrganelle(organelle){
 
         if(element.element.contains(selectedOrganelleListItem)){
             element.element.removeChild(selectedOrganelleListItem);
+            element.element.classList.remove("Selected");
```

Comment:
This method still has changes that I don't think are necessary. This logic is less foolproof